packages/browser/src/plugins/segmentio/normalize.ts

```diff
@@ -131,7 +131,8 @@ export function normalize(
   const ctx = json.context
 
   // This guard against missing ctx.page should not be neccessary, since context.page is always defined
-  const query: string = ctx.page?.search || ''
```

Comment:
Not sure where we set `search` for ourselves, but if it's internal maybe we should try putting our stuff in a particular nested object `internal`? or use `_search` ?